ast/ddl.go

```diff
@@ -1326,6 +1327,10 @@ func (n *TableOption) Restore(ctx *RestoreCtx) error {
 		ctx.WriteKeyWord("SHARD_ROW_ID_BITS ")
 		ctx.WritePlain("= ")
 		ctx.WritePlainf("%d", n.UintValue)
+	case TableOptionPreSplitRegion:
+		ctx.WriteKeyWord("PRE_SPLIT_REGIONS ")
+		ctx.WritePlain("= ")
+		ctx.WritePlainf("%d", n.UintValue)
```

Comment:
Let's combine this with the statement above.

```go
		ctx.WritePlainf("= %d", n.UintValue)
```

Reply:
done.